todo_app/app.py

```diff
@@ -1,11 +1,17 @@
-from flask import Flask
-
+from flask import Flask, render_template, request
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/', methods=['POST','GET'])
```

Comment:
Combining the POST & GET methods works, but can lead to some surprising behaviour because the POST directly returns the result; e.g. refreshing the page after submitting a new to-do tries to add the same one again. Breaking this out into two separate endpoints (one for GET and one for POST) and having the POST endpoint redirect (using the Flask-provided `redirect` function) to the GET endpoint on success would help with that